src/main.c

```diff
@@ -2732,6 +2731,12 @@ int main(int argc, char **argv) {
         // DATABASE INITIALIZATION //
         if (g->mode == MODE_OFFLINE || USE_CACHE) {
             db_enable();
+	    //---Requirement 1---
+	    //Simple test to see if database exists;
+	    //if not, generate a new world seed
+	    if (!(access(g->db_path, F_OK) == 0)){
```

Comment:
Would this be better served at the original seed generation location on line 2589? Leaving this to ensure a seed is generated inside the game loop is fine, but it may be useful to initialize outside the loop then use this to ensure it's working properly.

Reply:
Moving the seed generation to the initialization seems to generate a new seed every run.  I'll have to check if either the database or the model haven't been assigned at that point.